homeassistant/components/mvglive/sensor.py

```diff
@@ -40,7 +45,7 @@
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
-        vol.Optional(CONF_NEXT_DEPARTURE): [
+        vol.Required(CONF_NEXT_DEPARTURE): [
```

Comment:
I think this was a bug - what's the point of the integration if it's optional?

Reply:
hehe, it would not be helpful without that. It would even error :)